internal/app/init/pkg/system/system.go

```diff
@@ -138,3 +138,28 @@ func (s *singleton) List() (result []*ServiceRunner) {
 
 	return
 }
+
+// Stop will initiate a shutdown of the specified service.
+func (s *singleton) Stop(services ...Service) {
```

Comment:
should it accept service ids? not sure if it makes sense to construct the same instance of the service we pass to `Start()` (today service instances are very simple, but this might change in the future?)

Reply:
I did this just for parity with the `Start()` method. I think it'd be the same cost with how it's currently implemented ( still requiring a lookup against `service.List()` ) so I'm a bit indifferent. If you feel strongly I can change it.

Reply:
I don't think you need to lookup via `service.List()` as map keys are service IDs. Not feeling strong about it, but I think `service.List()` lookup could be avoided here at no cost.